examples/experimental/torch/classification/configs/inception_v3_imagenet_mixed_int_fracbits_msize.json

```diff
@@ -0,0 +1,46 @@
+{
+    "model": "inception_v3",
+    "pretrained": true,
+    "input_info": {
+      "sample_size": [2, 3, 299, 299]
+    },
+    "num_classes": 1000,
+    "batch_size" : 512,
+    "epochs": 1,
```

Comment:
I know that you set this by the same number of default configuration.
But, how this is going to be with more training epochs?

Reply:
As we can see in `README.md`, our target `compression_rate` was set `1.5` and it reached the target with this configuration. We can enlarge the training epochs for more `compression_rate` or the smaller initial bit-widths (e.g. `int4`).